src/Workleap.Extensions.Mongo.Tests/DistributedTracingTests.cs

```diff
@@ -39,7 +39,6 @@ public async Task Distributed_Tracing_On_Mongo_Commands_Works()
         var activities = recorder.RecordedActivities.ToArray();
         Assert.NotEmpty(activities);
         var activity = Assert.Single(activities, x => x.DisplayName == "things.insert");
-        this.Logger.LogInformation("{Count}", activities.Length);
```

Comment:
Personally, I decided to remove the log lines rather than refactor or add pragma exceptions. My reasoning is that logs would be useful during dev time or debugging but overall, we would not rely on them during our CI process

Reply:
Synced offline, these log lines in the tests are useful and should be kept for debugging purposes.